zerver/openapi/zulip.yaml

```
@@ -8961,8 +8961,8 @@ paths:
       summary: Mute a user
       tags: ["users"]
       description: |
-        This endpoint [mutes a user](/help/mute-a-user). Messages sent by users
-        you've muted will be automatically marked as read and hidden.
+        [Mute a user](/help/mute-a-user). Messages sent by muted users
+        will be automatically marked as read and hidden.
```

This wording sounds a lot like it's about setting a flag on the muted user globally. It'd be good to add a few words to clarify that it's only about a relationship between the user being muted and the user who's doing the muting.

Perhaps:
```suggestion
        [Mute a user](/help/mute-a-user) as seen by the requesting user.
        Messages sent by muted users will be automatically marked as read and hidden
        from the perspective of the user who muted them.
```

---

(The existing text isn't super clear about this either, but I think the "*you've* muted" helps)

---

Also added the "from the perspective of the requesting user" to the `api/unmute-user` endpoint description.